mm/mixtures.py

```diff
@@ -60,14 +60,14 @@ def inc_merge_count(e):
             util.partition_into_tree(mixtures_to_combine, combine_per_step),
         )
 
-        self.merge_log_file = self.log_file_output_path("merge", crp, merge_count)
-        self.mixtures = self.output_path("am.mix", cached=True)
+        self.out_merge_log_file = self.log_file_output_path("merge", crp, merge_count)
+        self.out_mixtures = self.output_path("am.mix", cached=True)
 
-        self.merge_rqmt = {"time": max(merge_count / 25, 0.5), "cpu": 1, "mem": 1}
```

Comment:
there are several Jobs that inherit from MergeMixturesJob (e.g. LinearAlignmentJob, EstimateMixturesJob), which have different rqmt for their working step and only do the merging after they are done. Their rqmt would overwrite this merging related rqmt if it were not called differently.